### DISCUSSION TITLE
TOC scrollspy에 관하여

### DISCUSSION THREAD

**Discussion by gia-world** [pronist/hELLO#123] [Q&A] [answered]
먼저 만들어 주신 스킨을 너무나도 잘 사용하고 있어 감사하다는 인사 드리고 싶습니다!
저는 퍼블리셔 2년차로, 세컨 블로그는 제가 직접 수정해보고 싶어 작업중인데요,
티스토리 스킨 자체도 조금 어렵지만 헬로스킨처럼 스크롤스파이를 사용하고 싶은데 적용이 어렵더라구요😥
일반 사이트처럼 id를 사용하는게 아니라 헤딩태그 기준이라... ㅠㅠ
혹시 플러그인을 사용한 것인지 아니면 직접 제작하신 건지 궁금합니다!
염치 불구하지만 스크롤스파이 적용할 때의 팁도 주시면 감사하겠습니당🙏

**Comment by pronist** [comment, marked as answer]
@gia-world 안녕하세요! 답변이 늦었네요.

Scrollspy 는 [UIKit](https://getuikit.com/) 에 포함된 라이브러리를 사용하였습니다. :)
